### ISSUE TITLE
Run `yarn start` in topological package order

### ISSUE THREAD

**Issue by wodeni** [penrose/penrose#482]
**Describe the bug**
When running `yarn start` on a fresh repo, the first execution will fail because `browser-ui` finishes running before rollup finishes. 
The known solution is to run `yarn start` again.

**To Reproduce**
1. Run `yarn clean`
2. Run `yarn`
3. Run `yarn start`
5. Run `yarn start` again

**Expected behavior**
`yarn start` works the first time around.

**Screenshots**
```
yarn run v1.22.10
$ lerna run watch --stream --parallel --scope=@penrose/core --scope=@penrose/browser-ui
lerna notice cli v3.22.1
lerna notice filter including ["@penrose/core","@penrose/browser-ui"]
lerna info filter [ '@penrose/core', '@penrose/browser-ui' ]
lerna info Executing command in 2 packages: "yarn run watch"
@penrose/browser-ui: $ CI=true react-scripts --max_old_space_size=8192 start
@penrose/core: $ yarn build:parsers
@penrose/core: $ nearleyc src/parser/Domain.ne > src/parser/DomainParser.ts && nearleyc src/parser/Substance.ne > src/parser/SubstanceParser.ts && nearleyc src/parser/Style.ne > src/parser/StyleParser.ts
@penrose/core: $ rollup -c -w
@penrose/core: loaded rollup.config.js with warnings
@penrose/core: (!) Unused external imports
@penrose/core: default imported from external module 'rollup-plugin-visualizer' but never used
@penrose/core: rollup v2.39.0
@penrose/core: bundles ./src/index.ts → build/dist/index.esm.js...
@penrose/browser-ui: ℹ ｢wds｣: Project is running at http://172.26.102.93/
@penrose/browser-ui: ℹ ｢wds｣: webpack output is served from
@penrose/browser-ui: ℹ ｢wds｣: Content not from webpack is served from /Users/wodeni/Code/penrose/packages/browser-ui/public
@penrose/browser-ui: ℹ ｢wds｣: 404s will fallback to /
@penrose/browser-ui: Starting the development server...
@penrose/browser-ui: Failed to compile.
@penrose/browser-ui: ./src/App.tsx
@penrose/browser-ui: Module not found: Can't resolve '@penrose/core' in '/Users/wodeni/Code/penrose/packages/browser-ui/src'
@penrose/browser-ui: Compiling...
@penrose/browser-ui: Failed to compile.
@penrose/browser-ui: ./src/App.tsx
@penrose/browser-ui: Module not found: Can't resolve '@penrose/core' in '/Users/wodeni/Code/penrose/packages/browser-ui/src'
@penrose/browser-ui: Compiling...
@penrose/browser-ui: Failed to compile.
@penrose/browser-ui: ./src/App.tsx
@penrose/browser-ui: Module not found: Can't resolve '@penrose/core' in '/Users/wodeni/Code/penrose/packages/browser-ui/src'
@penrose/browser-ui: Compiling...
@penrose/core: created build/dist/index.esm.js in 21.2s
@penrose/browser-ui: Failed to compile.
@penrose/browser-ui: /Users/wodeni/Code/penrose/packages/browser-ui/src/App.tsx
@penrose/browser-ui: TypeScript error in /Users/wodeni/Code/penrose/packages/browser-ui/src/App.tsx(13,8):
@penrose/browser-ui: Cannot find module '@penrose/core' or its corresponding type declarations.  TS2307
@penrose/browser-ui:     11 |   stateInitial,
@penrose/browser-ui:     12 |   stepUntilConvergence,
@penrose/browser-ui:   > 13 | } from "@penrose/core";
@penrose/browser-ui:        |        ^
@penrose/browser-ui:     14 |
@penrose/browser-ui:     15 | /**
@penrose/browser-ui:     16 |  * (browser-only) Downloads any given exported SVG to the user's computer
^C
```

**Additional context**
This is a known issue with rollup when used with lerna: https://github.com/rollup/rollup/issues/1919
Basically, only `--parallel` will start both react and rollup at the same time. Otherwise, lerna will respect the topological ordering, but rollup blocks the process.

**Comment by wodeni** [comment]
Closing for now as there's no obvious solution yet. Updated the wiki to require `yarn build` after first installation.
